docs/README.md

```diff
@@ -1,6 +1,38 @@
 # auth-service/Docs
 
 Documentation for the auth service
+## Database
+
+The following Figure depicts the Entity Relationship Diagram of the auth-service's schema:
+
+<p align="center">
+  <img src="http://www.plantuml.com/plantuml/proxy?cache=no&src=https://raw.githubusercontent.com/spikerheado1234/auth-service/master/docs/ErDiagram.puml">
+</p>
+
+The following table depicts the types of each attribute in the `Consent` table, algonside their constraints:
+
+|Attribute| Represented Internal Type | Constraints|
+|:---------:|:---------------------------:|:------------:|
+|`id`|VARCHAR(36)|Unique - Primary Key|
+|`initiator_id`|VARCHAR(36)|Non-Null|
+|`participant_id`|VARCHAR(36)|Non-Null|
+|`credential_id`|TEXT|(Can be Null)|
```

Comment:
Can we double check on the format of this? I suspect that this would be numeric, but we should probably limit this rather than leave it as an unbounded text field